perun-base/src/main/resources/perun-roles.yml

```diff
@@ -402,7 +402,19 @@ perun_policies:
     include_policies:
       - default_policy
 
-  getUserRoles_int_policy:
+  getUserRoles_int_boolean_policy:
+    policy_roles:
+      - PERUNOBSERVER:
+    include_policies:
+      - default_policy
+
+  getRoleComplementaryObjectsWithAuthorizedGroups_int_policy:
+    policy_roles:
+      - PERUNOBSERVER:
+    include_policies:
+      - default_policy
+
+  getRolesObtainedFromAuthorizedGroupMemberships_int_policy:
```

Comment:
These policies would not be enough for the roles view in "myProfile" section - every user can view his roles there. I think "`SELF: User`" policy needs to be added to the new policies.

Reply:
yes, they should